doc/examples/syn_registration_3d.py

```diff
@@ -42,7 +42,8 @@
 """
 
 from dipy.segment.mask import median_otsu
-stanford_b0_masked, stanford_b0_mask = median_otsu(data, median_radius=4, numpass=4)
+stanford_b0_masked, stanford_b0_mask = median_otsu(data, median_radius=4,
```

Comment:
This doesn't require a `vol_idx` input?

Reply:
No because the b0 itself is 3D. I accidentally introduced a typo. It should have been
stanford_b0_masked, stanford_b0_mask = median_otsu(stanford_b0, median_radius=4,
                                                   numpass=4)
syn_b0_masked, syn_b0_mask = median_otsu(syn_b0, median_radius=4, numpass=4)
I have corrected it.

Reply:
Gotcha. That makes sense!